### DISCUSSION TITLE
unauthorized activity

### DISCUSSION THREAD

**Discussion by SalluKasuri** [community/community#41029] [New to GitHub] [answered]
Dear GitHub Community, my account my GitHub account cannot work properly I think my account has some unauthorized activity can you please fix it and remove all unauthorized person that can interfere in my account 
msalmankasuri@gmail.com this is my personal Gmail account.
I cannot change my personal data it can show Sorry, something went wrong. Can you please fix it? I have not used my GitHub account.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @sallukasuri! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :) The Community is unable to assist so I'll be locking this post. Best of luck!
